src/test/scala/edu/ie3/simona/agent/participant/StorageAgentModelCalculationSpec.scala

```diff
@@ -522,7 +522,7 @@ class StorageAgentModelCalculationSpec
        */
 
       // Request flex options
```

Comment:
same here

Reply:
In this particular case I'd leave it as it is, because we're actually expecting flex options here